kong/runloop/handler.lua

```diff
@@ -1349,7 +1349,7 @@ return {
       end
 
       if var.upstream_http_upgrade and
-         var.upstream_http_upgrade ~= var.upstream_upgrade then
+         lower(var.upstream_http_upgrade) ~= var.upstream_upgrade then
```

Comment:
@bungle is `var.upstream_upgrade` guaranteed to be lowercase? since we're only one-sided lowering...

Reply:
@Tieske that's right.

Reply:
Though, as it is publicly accessible, perhaps it makes sense to do it on both sides.

Reply:
@Tieske changed to `lower` on both sides.